src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java

```diff
@@ -130,40 +109,16 @@ void testPostRequestServiceHearingValues() throws Exception {
     void testPostRequestServiceHearingValues_badCaseID() throws Exception {
```

Comment:
How did you get away with underscores in method names?  On ET PMD complained at us using them!
I do prefer to use them in unit test method names as it makes the unit test more readable :)

Reply:
I usually don't underscore them but I agree they can be nice :)

Reply:
Yeah I'm big into underscores as well for unit tests, we also use @DisplayName for tests which makes things a bit easier.